src/modules/dashboard/components/Extensions/Extensions.css

```diff
@@ -0,0 +1,7 @@
+.main {
+  max-width: 1000px;
```

Comment:
I'm not really sure about this, but since I couldn't really see how this looks on the page, leave it like this until the next round of reviews